### DISCUSSION TITLE
Could you get rid of the sign in requirement for NSFW?

### DISCUSSION THREAD

**Discussion by missionfloyd** [civitai/civitai#598] [Ideas]
Sure, hide/blur NSFW images by default, but don't make us make an account just to download models. It's really annoying how _everything_ these days insists you make an account to use it.

**Comment by Judas87Swiff** [comment]
"Think of the children" and "SEO" issues with ADs over NSFW content in general (Blur without account/sign-in gets by around this issue, just like how Patreon's mobile app completely blacklists any NSFW associated content creator from appearing in their mobile app search results), for some reason Patreon is fine with allowing the official website able to search for those NSFW people. Just food for thought.

**Reply by Efreak** [reply]
> Patreon's mobile app completely blacklists any NSFW associated content creator from appearing in their mobile app search results), for some reason Patreon is fine with allowing the official website able to search for those NSFW people

Blame this on Apple. I'm not sure if Google follows Apple on this these days, they certainly didn't used to. Google never allowed nsfw apps iirc, but I believe they allow apps with nsfw user-generated or Internet content as long as it's not visible in the screenshots and as long as nsfw specifically is not the intended purpose of the app. Same deal with torrent clients available in the play store: don't show screenshots of downloading movies and you're good.
